### DISCUSSION TITLE
Projects (beta) 'table' view doesn't show issue number

### DISCUSSION THREAD

**Discussion by 0xdevalias** [community/community#4716] [Projects and Issues] [answered]
Across most of GitHub the issue/PR number is sort of the canonical 'reference' that we refer to things by, yet in the new projects beta on the 'table' view it only shows the Issue/PR titles, not the issue number. This feels disjointed compared to the rest of the experience across GitHub, and makes it harder for us to refer to issues/PRs be a short canonical identifier.

**Comment by 700software** [comment]
Manual workaround: 😮 I added a text field and have typed in issue number #123 on each one.

(We work out of a single repo and others on the Zoom call need to know which issue number to refer to.)

**Comment by alexcnichols** [comment, marked as answer]
👋🏼 Improvements for displaying the issue number in the project table, and more, are coming soon! Check out this roadmap item for more and subscribe to be notified if/when things change. https://github.com/github/roadmap/issues/283 :octocat: 

Marking this as a duplicate, in favor of this other discussion:

- https://github.com/github/feedback/discussions/5904
